workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/services/rest/HeadlessCommerceAdminTaxonomy.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Liferay} from '../../liferay/liferay';
+import fetcher from '../fetcher';
+
+class HeadlessCommerceAdminTaxonomy {
```

Comment:
This file is not related to Commerce, rename the file as well

```suggestion
class HeadlessAdminTaxonomy {
```